docs/rules/no-duplicate-imports.md

```diff
@@ -14,57 +16,47 @@ import { find } from 'module';
 
 This inspection reports any duplicated module in an import statement.
```

Comment:
Suggest you move the “This rule requires …” sentence from above down here to replace this sentence.